demos/simple-menu.html

```diff
@@ -32,6 +32,14 @@
         flex-direction: column;
       }
 
+      .demo-header-toolbar {
+        z-index: 10;
+      }
+
+      #demo-menu {
```

Comment:
I was thinking this menu should _not_ overlap the toolbar since the toolbar is supposed to be thought of as being above the page.

If you want this to overlap the toolbar, it'd be simpler to only override the z-index on the menu in the hero section, instead of overriding the z-index of both the toolbar and this menu.

Also, generally avoid ID selectors in demo CSS.

Reply:
done